src/clientiface.cpp

```diff
@@ -242,7 +246,6 @@ void RemoteClient::GetNextBlocks (
 				FOV setting. The default of 72 degrees is fine.
 			*/
 
-			float camera_fov = (72.0*M_PI/180) * 4./3.;
```

Comment:
Yeah, no problem.
Random thought:
Since FOV is 1 value 'isBlockInSight' must be making an assumption, or using the larger horizontal FOV angle.